Add monetary price and currency

jannahastings opened the pull request:
Add 'monetary price' and 'currency' with the definitions 'A monetary price is a quantity value that describes the amount of money requested, expected, required or given in exchange for something else.' and 'A unit which is a measure of the medium of an exchange value, defined by reference to the geographical location of the monetary authorities responsible for it.' respectively as per the FIBO sheet. 

Closes #331

akleinau commented:
About the definition of monetary price: A monetary price is more concretely an economic value, quantity value is the superclass of economic value.

jannahastings commented:
> About the definition of monetary price: A monetary price is more concretely an economic value, quantity value is the superclass of economic value.

Good catch. Now fixed in definition.

sfluegel05 reviewed (changes requested):
You added a FIBO-reference for currency, but not for monetary price (https://spec.edmcouncil.org/fibo/ontology/FND/Accounting/CurrencyAmount/Price).

Also, could you update the changelog for this pr and the others as well?

jannahastings commented:
> You added a FIBO-reference for currency, but not for monetary price (https://spec.edmcouncil.org/fibo/ontology/FND/Accounting/CurrencyAmount/Price).

Reference now added. 

> Also, could you update the changelog for this pr and the others as well?

Changelog now updated.

sfluegel05 reviewed (approved):
Now everything looks fine to me. Thanks for implementing!
